package.json

```diff
@@ -7,9 +7,25 @@
     "husky": "^7.0.2",
     "lerna": "^4.0.0",
     "prettier": "~2.2.1",
-    "pretty-quick": "~3.1.0"
+    "pretty-quick": "~3.1.0",
+    "start-server-and-test": "^1.14.0"
   },
   "scripts": {
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "postinstall": "npm run bootstrap",
+    "bootstrap": "lerna bootstrap",
+    "build": "npm run build.web-components && npm run build.react & npm run build.angular & npm run build.astro-uxds",
+    "build.tests": "npm run build.stencil && npm run build.react",
+    "build.react": "npm --prefix packages/react run build",
```

Comment:
I like the idea, but I don't think it's a good idea to have a bunch of package-specific commands in our top level package, or the `npm --prefix` stuff. Just use [what Lerna has built in for filtering, etc.](https://www.npmjs.com/package/@lerna/filter-options) Like `build` could be `lerna run build`, while `build.tests` could be `lerna run --parallel --scope @(react|web-components) build`